tests/spark/test_spark_model_export.py

```diff
@@ -155,6 +198,21 @@ def test_model_export(spark_model_iris, model_path, spark_custom_env):
     assert spark_model_iris.predictions == preds3
     assert os.path.exists(sparkm.DFS_TMP)
 
+@pytest.mark.large
+def test_estimator_model_export(spark_model_estimator, model_path, spark_custom_env):
+    sparkm.save_model(spark_model_estimator.model, path=model_path, conda_env=spark_custom_env)
+    # score and compare the reloaded sparkml model
+    reloaded_model = sparkm.load_model(model_uri=model_path)
+    preds_df = reloaded_model.transform(spark_model_estimator.spark_df)
+    preds = [x.prediction for x in preds_df.select("prediction").collect()]
+    assert spark_model_estimator.predictions == preds
+
+
+@pytest.mark.large
+def test_transformer_model_export(spark_model_transformer, model_path, spark_custom_env):
+    with pytest.raises(MlflowException):
+        sparkm.save_model(spark_model_transformer.model, path=model_path, conda_env=spark_custom_env)
+
```

Comment:
Nit: Can we check that the error message contains some of the expected text, as we do in `test_sparkml_model_log_invalid_args`?